rag_apps/hybrid_rag_system/src/graph_indexer.py

```
@@ -0,0 +1,147 @@
+
+def _extract_from_chunk(client: Mistral, text: str) -> dict:
+    for attempt in range(3):
+        try:
+            resp = client.chat.complete(
+                model="mistral-small-latest",
+                messages=[{"role": "user", "content": _EXTRACT_PROMPT.format(text=text[:2500])}],
+                temperature=0.1,
+                max_tokens=800,
+            )
+            return _parse_extraction(resp.choices[0].message.content.strip())
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)